README.md

```diff
@@ -1,44 +1,88 @@
-# README
+# Calculate teacher pay
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> School teachers and leaders in England can use this service to see how their
+  salary could change following the 2023 pay award.
 
-Things you may want to cover:
+This repository contains the source code and infrastructure definitions for the
+'Calculate teacher pay' service, a minimalistic Ruby on Rails application.
 
-* Ruby version
+## Local development setup
 
-* System dependencies
+After cloning the repository, `cd` into the folder and install the dependencies with:
 
-* Configuration
+```
+$ yarn install
+$ bundle install
+$ bundle exec foreman start
+```
 
-* Database creation
+Finally, visit: http://127.0.0.1:3000/
 
-* Database initialization
+## Testing
 
-* How to run the test suite
+To run the unit test suite locally:
 
-* Services (job queues, cache servers, search engines, etc.)
+```
+$ bundle exec rspec
+```
 
-* Deployment instructions
+To perform manual testing against the hosted `development` enviornment, e.g.
+when testing a dependency update, follow the instruction below (see [Deployment](#deployment)).
 
-* ...
+Please note: the hosted `development` environment is deployed with
+`RAILS_ENV=production`.
 
-## Setup
+## Deployment
 
-### Prerequisites
+There currently are two hosted environments in AKS, `development` and
+`production`, and the respective urls are:
 
-This project depends on:
+- https://development.calculate-teacher-pay.education.gov.uk/
+- https://calculate-teacher-pay.education.gov.uk/
 
-  - [Ruby](https://www.ruby-lang.org/)
-  - [Ruby on Rails](https://rubyonrails.org/)
-  - [NodeJS](https://nodejs.org/)
-  - [Yarn](https://yarnpkg.com/)
-  - [Postgres](https://www.postgresql.org/)
+Every merge to `main` will trigger a deployment to `development` and
+`production` via a Github Action workflow.
 
-### Linting
+In some circumstances, you may need to trigger a manual deployment to
+`development` first, e.g. to test a new feature before it's ready for release.
 
-To run the linters:
+To trigger a manual deployment, you can use the Github Action workflow
+"Deploy to AKS (Manual)", specifying a target docker image tag and environment.
```

Comment:
We could mention the default tag, which should be `main` if we fix it here: https://github.com/DFE-Digital/teacher-pay-calculator/blob/main/.github/workflows/manual-deploy.yml#L10